src/otx/cli/cli.py

```diff
@@ -431,18 +469,42 @@ def _prepare_subcommand_kwargs(self, subcommand: str) -> dict[str, Any]:
             k: v for k, v in self.config_init[subcommand].items() if k in self._subcommand_method_arguments[subcommand]
         }
 
-    def save_config(self) -> None:
+    def save_config(self, work_dir: Path) -> None:
         """Save the configuration for the specified subcommand.
 
+        Args:
+            work_dir (Path): The working directory where the configuration file will be saved.
+
         The configuration is saved as a YAML file in the engine's working directory.
         """
+        self.config[self.subcommand].pop("workspace", None)
         self.get_subcommand_parser(self.subcommand).save(
             cfg=self.config.get(str(self.subcommand), self.config),
-            path=Path(self.engine.work_dir) / "configs.yaml",
+            path=work_dir / "configs.yaml",
             overwrite=True,
             multifile=False,
             skip_check=True,
         )
+        # if train -> Update `.cache` folder
+        if self.subcommand == "train":
+            self.update_cache(work_dir=work_dir)
+
+    def update_cache(self, work_dir: Path) -> None:
+        """Update the cache directory with the latest configurations and checkpoint file.
+
+        Args:
+            work_dir (Path): The working directory where the configurations and checkpoint files are located.
+        """
+        cache_dir = work_dir.parent / ".cache"
+        cache_dir.mkdir(exist_ok=True)
+        with (cache_dir / "configs.txt").open("w") as f:
+            f.write(str((work_dir / "configs.yaml").resolve()))
+        checkpoint_dir = work_dir / "checkpoints"
+        ckpt_list = list(checkpoint_dir.glob("epoch_*.ckpt"))  # Convert generator to list
+        if ckpt_list:
+            latest_checkpoint_file = max(ckpt_list, key=lambda p: p.stat().st_mtime)
+            with (cache_dir / "latest_checkpoint.txt").open("w") as f:
+                f.write(str(latest_checkpoint_file.resolve()))
```

Comment:
Why not just creating a soft link for it?

Reply:
Thanks, Done - https://github.com/openvinotoolkit/training_extensions/pull/2934/commits/6b560edf42ebdfdbf9bc3268df6adcb375a47f07